microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md

```diff
@@ -3,62 +3,140 @@ description: The process of distributing extension by mechanism other than verif
 title: Alternate Method of Distributing Extension
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 09/15/2020
+ms.date: 02/04/2021
 ms.topic: article
 ms.prod: microsoft-edge
-keywords: edge-chromium, extensions development, browser extensions, addons, partner center, developer
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer
 ---
+# Alternative extension distribution methods  
 
-# Alternate Method of Distributing Extension  
+All Microsoft Edge extensions must be distributed using one of two methods.
```

Comment:
@reeza It's a standard terminology which we use for extensions. Can we retain the title as the previous one?